<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertBrawlerAircraftDialog</class>
 <widget class="QWidget" name="ConvertBrawlerAircraftDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Brawler Converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="mDialogGraphic">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources/icons.qrc">:/icons/Dark/64x64/aircraft.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Convert Brawler Aircraft</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
       <widget class="QGroupBox" name="fileBrowseGroupBox">
         <property name="title">
           <string>Source File</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
             <widget class="QLineEdit" name="mInputFileLineEdit">
               <property name="text">
                 <string>Pick a Brawler .FXW file</string>
               </property>
             </widget>
           </item>
           <item row="0" column="1">
             <widget class="QPushButton" name="mFileBrowseButton">
               <property name="text">
                 <string>Browse</string>
               </property>
             </widget>
           </item>
         </layout>
       </widget>
     </item>
     <item row="1" column="0">
       <widget class="QGroupBox" name="newVehicleGroupBox">
         <property name="title">
           <string>New Aircraft Name</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
             <widget class="QLineEdit" name="mInstanceNameLineEdit">
               <property name="minimumSize">
                 <size>
                   <width>0</width>
                   <height>26</height>
                 </size>
               </property>
             </widget>
           </item>
         </layout>
       </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="mOkButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mCancelButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
